src/garage/experiment/deterministic.py

```diff
@@ -15,13 +16,21 @@ def set_seed(seed):
 
     """
     seed %= 4294967294
+    # pylint: disable=global-statement
     global seed_
     seed_ = seed
     random.seed(seed)
     np.random.seed(seed)
     if 'tensorflow' in sys.modules:
-        import tensorflow as tf
+        import tensorflow as tf  # pylint: disable=import-outside-toplevel
         tf.compat.v1.set_random_seed(seed)
+    if 'torch' in sys.modules:
+        logger.log('Enabeling deterministic mode in pytorch mode can have a \
```

Comment:
`warnings.warn` is a better way for this right now. the logger might not yet have a StdOutput

Reply:
also, if this only affects GPU, please say so.

Reply:
Using `warnings.warn` now.